features/hook.feature

```diff
@@ -71,3 +71,34 @@ Feature: Tests `WP_CLI::add_hook()`
       `add_hook()` to the `before_invoke` is working.
       """
     And the return code should be 0
+
+  Scenario: Add callback to the `before_run_command` with args
+    Given a WP installation
+    And a before-run-command.php file:
+      """
+      <?php
+      $callback = function( $args, $assoc_args, $options ) {
```

Comment:
The parentheses should directly follow the function name, but `function` is not the function name and we're not "executing" the `function` keyword.

There should be 1 space between the `function` keyword and the function name (which is empty), and then no space between the function name and the parentheses.
```suggestion
      $callback = function ( $args, $assoc_args, $options ) {
```